vendor/profile.ps1

```diff
@@ -74,7 +74,7 @@ $env:Path = "$Env:CMDER_ROOT\bin;$env:Path;$Env:CMDER_ROOT"
 $CmderUserProfilePath = Join-Path $env:CMDER_ROOT "config/user-profile.ps1"
 if(Test-Path $CmderUserProfilePath) {
     # Create this file and place your own command in there.
-    Invoke-Expression $CmderUserProfilePath
+    & $CmderUserProfilePath
```

Comment:
If that's to just run the profile.ps1 into the current session you want to dot source it:

`. "CmderUserProfilePath"`

Honestly I'm not sure what `&` will do different but you'll see a lot of 'dot source' when looking at powershell examples so it would be more consistent.